server/routes/connector_status.py

```
@@ -516,6 +516,61 @@ def _check_pagerduty(creds: Dict[str, Any]) -> Dict[str, Any]:
+        r = requests.get(
+            f"{endpoint}/_cluster/health",
+            auth=(username, password),
+            timeout=HTTP_TIMEOUT,
+            verify=creds.get("verify_ssl", True),
+        )
+        if r.status_code in (401, 403):
+            return {"connected": False}
+        if r.ok:
+            return {"connected": True, "endpoint": endpoint}
+    except Exception:
+        pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)